core/common/assets/js/api/core/hooks/data.js

```diff
@@ -27,9 +27,14 @@ export default class Data extends HooksBase {
 			}
 
 			case 'catch':
-			case 'after': {
-				return callback.callback( args, result ) || true;
-			}
+			case 'after':
+				const callbackResult = callback.callback( args, result );
+
+				if ( ! callbackResult ) {
+					return true;
```

Comment:
true?

Reply:
Yes if you change it to false, tests will fall.

Reply:
Yes if you change it to false, tests will fall.

Reply:
Try find a solution.

Reply:
Handled.